cypress/support/commands.ts

```diff
@@ -17,11 +21,24 @@ const seed = Cypress.env('testSeed')
 const publicKey = Cypress.env('testPublicKey')
 const ethereumApi = Cypress.env('REACT_APP_UNCHAINED_ETHEREUM_HTTP_URL')
 const bitcoinApi = Cypress.env('REACT_APP_UNCHAINED_BITCOIN_HTTP_URL')
+const wsUrl = Cypress.env('wsUrl')
 const _0xApi = Cypress.env('0xApi')
 const coinGeckoApi = Cypress.env('coinGeckoApi')
 const foxContract = Cypress.env('foxContract')
 
 const walletDb = getWalletDbInstance()
+const ethTransaction = makeEthTxHistory()
```

Comment:
As we are returning multiple ETH transactions, should this be plural?

`ethTransactions`